.circleci/config.yml

```diff
@@ -177,6 +257,19 @@ workflows:
       - request-publish-dev-cloud:
           type: approval
           filters: *upstream-feature-branch
+      
+      - build-and-publish-docker-arm:
```

Comment:
have you had a chance to try cross-compilation like what is mentioned here: 
https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/
https://blog.siposdani87.com/posts/multi-platform-docker-image-build-on-aarch64.html 
?
that might save us some CI code and build both things together.

In these examples they use go builds, which can totally cross-compile the code. In this case you get the python for the target architecture... and then it has to run to do the work. I suppose that might be a problem, because there's no virtualization layer?..

Reply:
Yes, I tried it, and ran into problems cross-compiling TF2 for aarch64 on x86_64/amd64. Which is why I am doing ARM image building on ARM hardware.